Week-2/Exercise 2.1/css/style.css

```diff
@@ -0,0 +1,268 @@
+*,
+*::after,
+*::before {
+    box-sizing: border-box;
+}
+
+:root {
+    --primary-color: rgb(145, 234, 122);
+    --secondary-color: rgb(181, 232, 169);
+}
+
+/*general styling */
+body {
+    font-family: "Poppins", sans-serif;
+    height: 100vh;
+    line-height: 1.6;
+    padding: 1em;
+}
+
+ul {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+    font-size: 1.3rem;
+}
+
+input,
+button {
+    font: inherit;
+}
+
+/*header styling*/
+
+.menu {
+    color: white;
+    cursor: pointer;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+}
+
+nav {
+    display: none;
+}
+
+nav a {
+    color: white;
+}
+
+nav>ul {
+    display: flex;
+    gap: 3rem;
+}
+
+ul>li:hover {
```

Comment:
space between
ul > li: hover

Reply:
Code formatter (ESLint) was removing the spaces. Will try to figure this out.